test_scripts/WebEngine/WebEngineProjection/013_Show_RPC_to_Main_Window_widget_without_DuplicateID.lua

```diff
@@ -0,0 +1,57 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description: Processing of RPCs for widgets which do not duplicate main window
+-- "duplicateUpdatesFromWindowID" param is not defined) when application has active WEB_VIEW template
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) WebEngine App with WEB_VIEW HMI type is registered
+-- 3) App successfully created a widget
+-- 4) Widget is activated on the HMI and has FULL level
+-- Steps:
+-- 1) App send Show(with WindowID for Main window) request to SDL
+-- SDL does:
+--  - send request UI.Show(without WindowID) to HMI
+-- 2) HMI sends UI.Show response "SUCCESS"
+-- SDL does:
+--  - send Show response with (success: true resultCode: "SUCCESS") to App
+--  - not send OnSystemCapabilityUpdated notification to App
+---------------------------------------------------------------------------------------------------
+--[[ General test configuration ]]
+config.defaultMobileAdapterType = "WS"
+
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = { "WEB_VIEW" }
+
+local createWindowParams = {
+  windowID = 1,
+  windowName = "Name",
+  type = "WIDGET"
+}
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = appHMIType
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
```

Comment:
I would propose to backmerge current feature branch and use `common.setupRAIParams` function for this.
This relates to other test scripts.

Reply:
Done in [(173089)](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/33/commits/173089eb6bf1a528d8f509b7f7f2916ee1b1e1d8)